src/lando/pulse/management/commands/pulse_notify.py

```diff
@@ -0,0 +1,87 @@
+import argparse
+from typing import Optional
+
+from django.core.management.base import BaseCommand, CommandError, CommandParser
+
+from lando.main.models.repo import Repo
+from lando.pulse.pulse import PulseNotifier
+from lando.pushlog.models.push import Push
+
+
+class Command(BaseCommand):
+    help = "Send Pulse notification for selected Pushes"
+    name = "pulse_notify"
+
+    def add_arguments(self, parser: CommandParser):
+        parser.add_argument("-r", "--repo", required=True)
+        parser.add_argument("-p", "--push-id", type=int, default=0)
+        parser.add_argument(
+            "-R",
+            "--force-renotify",
+            default=False,
+            action=argparse.BooleanOptionalAction,
+        )
+        parser.add_argument(
+            "-O",
+            "--force-out-of-order",
+            default=False,
+            action=argparse.BooleanOptionalAction,
+        )
+
+    def handle(self, *args, **options) -> None:
+        repo_name = options["repo"]
+        push_id = options.get("push_id")
```

Comment:
nit: This also technically defaults to `0` no?

Reply:
Ah! Yes!